en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-310.md

```diff
@@ -2116,7 +2116,95 @@ Upgrade the WSO2 API Manager Analytics database from version 3.0.0 to version 3.
     ALTER TABLE APILASTACCESSSUMMARY DROP CONSTRAINT APILASTACCESSSUMMARY_pkey;
     ALTER TABLE APILASTACCESSSUMMARY ADD PRIMARY KEY (APINAME,APICREATOR,APIVERSION,APICREATORTENANTDOMAIN);
     ```
-
+!!! note
+    Type and name of a column of few tables were changed through WUM in version 2.6. It is important to add the above
+    change into your database prior to migration. So execute the below query to 
+    confirm whether the above change is already available in your system.
+    
```

Comment:
Isn't it better to check whether the column name is already updated or not and ALTER table only if it was not updated yet?
Then we can add these queries to the above set of queries too, without running separately.

Reply:
This will be an improvement and will be addressed via another PR.